editor/src/clj/editor/build_errors_view.clj

```diff
@@ -196,10 +196,9 @@
         style (case (:severity error-item)
                 :info #{"severity-info"}
                 :warning #{"severity-warning"}
-                #{"severity-error"})
-        image (icons/get-image-view icon 16)
-        text (Text. message)]
-    {:graphic (HBox. (ui/node-array [image text]))
+                #{"severity-error"})]
+    {:text message
+     :icon icon
```

Comment:
Not sure why there was a `:graphic` with `Label` and `ImageView` when `:text` and `:icon` achieve the same, and the CSS styling is more efficient. I haven't noticed any changes in the build errors view.